.github/workflows/kakarot_rpc.yml

```diff
@@ -1,5 +1,5 @@
 ---
-name: RPC
+name: release
```

Comment:
If we change this, shouldn't we change the name of the file too? `_release` instead of `_rpc`?